README.md

```diff
@@ -19,6 +19,18 @@ available at `transit/`
 Development
 -----------
 
+### Environment Setup
+Create an AWS profile called `geotrellis-site`
+```bash
+aws --profile geotrellis-site configure
+```
+
+Create and provision the VM
+```bash
+$ vagrant up --provision
```

Comment:
I think that it would be good to replace this step with the usual `setup` script that does mostly this.